### PR TITLE
Strings of formatter class were replaced by constants from interface

### PR DESCRIPTION
**Pull request by metaaa**
New EnhancedButtonInterface were created and strings of formatter class were replaced by constants.

### CODE REVIEW

File: src/Plugin/EnhancedButtonInterface.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\enhanced_button_link\Plugin;
```
Comment (olunov) [inline src/Plugin/EnhancedButtonInterface.php]: Move EnhancedButtonInterface to namespace Drupal\enhanced_button_link.
File EnhancedButtonInterface.php should be placed to src directory (level up)

File: src/Plugin/Field/FieldFormatter/EnhancedButtonFormatter.php [hunk 2]
```diff
@@ -9,6 +9,7 @@
 use Drupal\Core\Utility\Token;
 use Drupal\link\Plugin\Field\FieldFormatter\LinkFormatter;
 use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\enhanced_button_link\Plugin\EnhancedButtonInterface;
```
Comment (olunov) [inline src/Plugin/Field/FieldFormatter/EnhancedButtonFormatter.php]: update namespace after moving EnhancedButtonInterface

File: src/Plugin/EnhancedButtonInterface.php (hunk 1 above)
Comment (metaaa) [reply src/Plugin/EnhancedButtonInterface.php]: done

File: src/Plugin/Field/FieldFormatter/EnhancedButtonFormatter.php (hunk 2 above)
Comment (metaaa) [reply src/Plugin/Field/FieldFormatter/EnhancedButtonFormatter.php]: done

**Pull request merged**